components/figma/actions/delete-a-comment/delete-a-comment.mjs

```diff
@@ -0,0 +1,56 @@
+import figmaApp from "../../figma.app.mjs";
+
+export default {
+  name: "Delete a Comment",
+  description: "Delete a comment to a file. [See the docs here](https://www.figma.com/developers/api#delete-comments-endpoint)",
+  key: "figma-delete-a-comment",
```

Comment:
We usually omit the articles in the component key/name, so the component directory and filename should also be changed

```suggestion
  key: "figma-delete-comment",
```